index.html

```diff
@@ -45,6 +45,34 @@ <h2 class="font2">I'm a Software Developer</h2>
                 <li><a href="https://medium.com/"><img src="assets/mobile/medium.svg" alt="medium icon that will link to Medium page"></a></li>
             </ul>
         </section>
+        <section class="modal modal-close">
+            <div class="modal-container d-flex">
+                <div class="modal-title d-flex">
+                        <div class="name-container d-flex">
+                            <h3 class="name title">Multi Post Stories</h3>
+                            <img src="assets/mobile/close-Icon.svg" alt="close button" class="close-button">
+                        </div>
+                        <ul class="tech-container skill-lists border-1">
+                            <li>HTML</li>
+                            <li>Bootstrap</li>
+                            <li>Ruby on Rails</li>
+                        </ul>
+                </div>
+                <div class="modal-body d-flex">
+                    <img class="modal-image"  src="assets/mobile/ImgPlaceholder-desktop.jpg" alt="image placeholder desktop">
+                    <div class="project-description-container">
+                        <div class="project-description d-flex font-body-reg">
+                            <p class="prj-description">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s.</p>
+        
+                        </div>
+                        <ul class="links d-flex">
+                            <li><a class="button d-flex see-live"><span>See live</span><img class="img-links" src="assets/mobile/Icon-seelive.svg" alt="see live icon"></a></li>
+                            <li><a class="button d-flex see-source"><span>See source</span><img class="img-links" src="assets/mobile/see-source-icon.svg" alt="see source icon"></a></li>
+                        </ul>
+                    </div>   
+                </div>
+            </div>
+        </section>
         <section class="work-section" id="portfolio">
```

Comment:
<img width="791" alt="DSC_2740-removebg-preview" src="https://github.com/Kccendana/portfolio_project/assets/103386183/5bff73cd-00cc-4318-b228-eaa788111d50">

Kindly implement the instructions stated in the requirement as shown above. All the project sections should be implemented dynamically with JavaScript.👆